components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const Button = ({ setProduct, produkt, productToBuy }) => {
+  return (
+    <StyledButton
+      onClick={() => {
+        setProduct(produkt);
```

Comment:
Update produkt to product in all lines.
```suggestion
        setProduct(product);
```